src/filters.js

```diff
@@ -231,6 +231,13 @@ var filters = {
         return str.toLowerCase();
     },
 
+    nl2br: function(str) {
+        if ((str === null) || (str === undefined)) {
+            return '';
+        }
+        return r.copySafeness(str, str.replace(/\r\n|\r|\n/g, '<br />\n'));
```

Comment:
"Empty Elements: Include a space before the trailing / and > of empty elements, e.g. <br />" cf. https://www.w3.org/TR/xhtml1/#guidelines

But this was ages ago. With the improved state of the current browsers, the space can be safely removed nowadays. I did put the space to maximize the chance of this PR being accepted :-) So let me know whether you want the space of not.